modules/install-creating-aws-local-zones-vpc-stack.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+[id="install-aws-local-zones-vpc-stack_{context}"]
+=== Creating a VPC for AWS Local Zones
+
+Create a virtual private cloud (VPC) to use with AWS Local Zones.
+
+.Procedure
+// TODO: GitHub links are not okay.
+. From a command line, enter the following command to create the stack:
++
+[source,terminal]
+----
+INSTALLER_URL="https://raw.githubusercontent.com/openshift/installer/master"
+TPL_URL="${INSTALLER_URL}/upi/aws/cloudformation/01_vpc.yaml"
+
+aws cloudformation create-stack \
+    --region ${AWS_REGION} \
+    --stack-name ${CLUSTER_NAME}-vpc \
+    --template-body ${TPL_URL} \
+    --parameters \
+        ParameterKey=VpcCidr,ParameterValue=${VPC_CIDR} \
+        ParameterKey=SubnetBits,ParameterValue=${VPC_SUBNETS_BITS} \
+        ParameterKey=AvailabilityZoneCount,ParameterValue=${VPC_SUBNETS_COUNT}
+----
+
+. Enter the following command to wait for the stack creation to finish:
++
+[source,terminal]
+----
+aws cloudformation wait stack-create-complete \
+    --region ${AWS_REGION} \
+    --stack-name ${CLUSTER_NAME}-vpc 
+----
+// TODO: jq is not okay.
```

Comment:
all `jq` references have been replaced on the installer PR, please update it

- https://github.com/openshift/installer/pull/6993
- https://github.com/openshift/installer/blob/master/docs/user/aws/install_existing_vpc_local-zones.md